src/lycophron/app.py

```diff
@@ -30,52 +34,97 @@ def __call__(cls, *args, **kwargs):
 
 
 class LycophronApp(object, metaclass=SingletonMeta):
```

Comment:
`init` made me rework a lot of the  `LycophronApp` class to allow more flexible instantiation. 

e.g. 

- `lycophron init` assumes the current directory
- `lycophron init --force` , same as before
- `lycophron init monkeys` creates a sub-directory `monkeys`

To allow this, the configuration variables had to be dynamic as well (e.g. to create the configuration files correctly, set the `DB_URI` correctly)